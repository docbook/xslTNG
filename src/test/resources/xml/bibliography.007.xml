<?xml version="1.0" encoding="utf-8"?>
<book xmlns="http://docbook.org/ns/docbook"
      version="5.0">
  <?db bibliography-collection="../bibcollection.xml"?>
  <info>
    <title>Unit Test: bibliography.007.xml</title>
    <abstract>
      <para>This tests not-“auto” bibliographies.</para>
    </abstract>
  </info>

  <chapter>
    <title>Some title</title>

    <para>Citations: <citation>AhoSethiUllman96</citation> and <citation>XML</citation>.</para>

    <para>Biblioref: <biblioref linkend="bib.xml"/>.</para>

    <para>The bibliography should contain cited works and the
    unreferenced entry “Walsh96”.</para>
  </chapter>

  <bibliography>
    <bibliomixed xml:id="bib.xml"/>
    <biblioentry>
      <abbrev>AhoSethiUllman96</abbrev>
      <authorgroup>
         <author>
            <personname>
               <firstname>Alfred V.</firstname>
               <surname>Aho</surname>
            </personname>
         </author>
         <author>
            <personname>
               <firstname>Ravi</firstname>
               <surname>Sethi</surname>
            </personname>
         </author>
         <author>
            <personname>
               <firstname>Jeffrey D.</firstname>
               <surname>Ullman</surname>
            </personname>
         </author>
      </authorgroup>
      <title>Compilers, Principles, Techniques, and Tools</title>
      <publisher>
         <publishername>Addison-Wesley Publishing Company</publishername>
      </publisher>
      <copyright>
         <year>1996</year>
         <holder>Bell Telephone Laboratories, Inc.</holder>
      </copyright>
      <biblioid class="isbn">0-201-10088-6</biblioid>
      <editor>
         <personname>
            <firstname>James T.</firstname>
            <surname>DeWolf</surname>
         </personname>
      </editor>
   </biblioentry>
   <bibliomixed xml:id="Walsh96">
      <bibliomset relation="article">
         <personname>
            <surname>Walsh</surname>
         </personname>, <personname>
            <firstname>Norman</firstname>
         </personname>.
    <title role="article">Introduction to Cascading Style Sheets</title>.
  </bibliomset>
      <bibliomset relation="journal">
         <title>The World Wide Web Journal</title>.
    <volumenum>2</volumenum>(<issuenum>1</issuenum>).
    <publishername>O'Reilly &amp; Associates, Inc.</publishername> and
    <orgname>The World Wide Web Consortium</orgname>.
    <pubdate>Winter, 1996</pubdate>
      </bibliomset>.
</bibliomixed>
</bibliography>
</book>
