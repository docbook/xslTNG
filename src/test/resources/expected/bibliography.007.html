<!DOCTYPE html><html xmlns="http://www.w3.org/1999/xhtml" class="no-js"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8"/><script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script><title>Unit Test: bibliography.007.xml</title><meta name="viewport" content="width=device-width, initial-scale=1.0"/><link href="https://purl.org/dc/elements/1.1/" rel="schema.dc"/><meta content="2011-04-22T17:02:00-06:00" name="dc.modified"/><meta content="DocBook xslTNG" name="generator"/><link href="./css/docbook.css" rel="stylesheet" media="screen"/></head><body class="home"><nav class="top"></nav><main><article class="book division"><header><h1>Unit Test: bibliography.007.xml</h1><div class="abstract"><h3 class="title">Abstract</h3><p>This tests not-“auto” bibliographies.</p></div></header><div class="list-of-titles"><div class="lot toc"><div class="title">Table of Contents</div><ul class="toc"><li><a href="#R_ch1"><span class="label">1</span><span class="sep">. </span>Some title</a></li><li><a href="#R_bi1">Bibliography</a></li></ul></div></div><section id="R_ch1" class="chapter component"><header><h2>Chapter <span class="label">1</span><span class="sep">. </span>Some title</h2></header><p>Citations: <a href="#R_bi1_biblioentry1" class="xref xref-biblioentry"><span class="label"><span class="abbrev">AhoSethiUllman96</span></span></a> and <a href="#bib.xml" class="xref xref-bibliomixed"><span class="label"><span class="abbrev">XML</span></span></a>.</p><p>Biblioref: <a href="#bib.xml" class="xref xref-bibliomixed"><span class="label"><span class="abbrev">XML</span></span></a>.</p><p>The bibliography should contain the cited works and the
    unreferenced entry “Walsh96”.</p></section><div id="R_bi1" class="bibliography component"><header><h2>Bibliography</h2></header><p id="bib.xml" class="bibliomixed">[<span class="abbrev">XML</span>] Tim Bray,
Jean Paoli, C. M. Sperberg-McQueen, Eve Maler, and François Yergeau, editors.
<cite class="citetitle"><a href="https://www.w3.org/TR/xml" class="link">Extensible Markup
Language (XML) 1.0 (Fifth Edition)</a></cite>.
World Wide Web Consortium, November 2008.</p><p id="R_bi1_biblioentry1" class="biblioentry">[<span class="abbrev">AhoSethiUllman96</span>] <cite class="title">Compilers, Principles, Techniques, and Tools</cite>. <span class="authorgroup"><span class="author"><span class="first-last personname"><span class="firstname">Alfred V.</span> <span class="surname">Aho</span></span></span>, <span class="author"><span class="first-last personname"><span class="firstname">Ravi</span> <span class="surname">Sethi</span></span></span>, and <span class="author"><span class="first-last personname"><span class="firstname">Jeffrey D.</span> <span class="surname">Ullman</span></span></span></span>. <span class="copyright">Copyright © <span class="copyright-years"><span class="year">1996</span></span> <span class="holder">Bell Telephone Laboratories, Inc.</span></span>. <span class="editor"><span class="first-last personname"><span class="firstname">James T.</span> <span class="surname">DeWolf</span></span></span>. <span class="biblioid">0-201-10088-6</span>. <span class="publishername">Addison-Wesley Publishing Company</span>. </p><p id="Walsh96" class="bibliomixed">[Walsh96] 
      <span class="bibliomset">
         <span class="first-last personname"><span class="surname">Walsh</span></span>, <span class="first-last personname"><span class="firstname">Norman</span></span>.
    <q class="article title">Introduction to Cascading Style Sheets</q>.
  </span>
      <span class="bibliomset">
         <cite class="title">The World Wide Web Journal</cite>.
    <span class="volumenum">2</span>(<span class="issuenum">1</span>).
    <span class="publishername">O'Reilly &amp; Associates, Inc.</span> and
    <span class="orgname">The World Wide Web Consortium</span>.
    <time class="pubdate">Winter, 1996</time>
      </span>.
</p></div></article></main><nav class="bottom"></nav></body></html>